PennCourses/settings/base.py

```diff
@@ -56,6 +57,18 @@
     'review',
 ]
 
+#From labs-accounts
+AUTHENTICATION_BACKENDS = [
+    'accounts.backends.LabsUserBackend',
+    'django.contrib.auth.backends.ModelBackend',
+]
+
+#From labs-accounts
+PLATFORM_ACCOUNTS = {
+    'REDIRECT_URI': os.environ.get('LABS_REDIRECT_URI', 'https://penncourseplan.com/accounts/callback/'),
```

Comment:
Hmm, we have a weird issue where this one instance of Django is serving PCA and PCP. I'm not a huge fan of just hardcoding the URI to be for PCP because then PCA logins will route though PCP, but I'm also not sure of a good alternative. @davish thoughts?

Reply:
We haven't set up `api.penncourses.org` yet, but this might be a good way to do it. Although I think for the PCP launch it's fine to send it thru PCP, since PCA isn't using accounts yet.

Reply:
Also though, we would also want this to be different in dev and prod, right? redirect to `localhost/accounts/callback/` in dev? or does it not work like that

Reply:
You can change the env variable to set it to localhost in dev

Reply:
So eventually we should change that uri to api.penncourses.org?

Reply:
Ok I squashed the commits.

Reply:
@davish I'll finalize some of the details of using accounts in development and add the configuration to the django template and here.